File: WebSport24hNews/Application/Query/Model/Account/User24hQuery.cs
namespace WebSport24hNews.Application.Query.Model.Account
{
    public class User24hQuery
    {
        public decimal Id { get; set; }

        public string? Username { get; set; }

        public string? Password { get; set; }

        public bool? Inactive { get; set; }

        public string? Role { get; set; }

        public string? Phone { get; set; }

        public string? Fullname { get; set; }

        public DateTime? Modifydate { get; set; }

        public string? Modifyby { get; set; }

        public DateTime? Createdate { get; set; }
        public string? Createby { get; set; }

        public string? Tokenactive { get; set; }

        public DateTime? Dateactive { get; set; }

        public string? Codeactive { get; set; }

        public DateTime? Datecodeactive { get; set; }

        public string? Center { get; set; }

        public bool? Addgoogle { get; set; }

        public decimal? Empid { get; set; }
    }
}

File: clear-all/clear-all/src/main/java/com/example/clear_all/dto/response/User24hQuery.java
package com.example.clear_all.dto.response;


import lombok.Data;

import java.math.BigDecimal;
import java.math.BigInteger;
import java.util.Date;

@Data
public class User24hQuery {
    private BigDecimal id;
    private String username;
    private String fullname;
    private String phone;
    private String role;
    private Short inactive;
    private Date createdate;
    private String createby;
    private Date modifydate;
    private String modifyby;
    private String tokenactive;
    private Date dateactive;
    private String codeactive;
    private Date datecodeactive;
    private String center;
    private Short addgoogle;
    private BigInteger empid;

}
